src/pages/Dashboard/Modals/SupplyWithdraw/index.stories.tsx

```diff
@@ -42,20 +42,6 @@ DisconnectedSupply.args = {
   isWithdrawLoading: false,
 };
 
-export const DisabledSupply = Template.bind({});
```

Comment:
Why are we removing this story?

Reply:
Because I forgot to make this update ahah
This story is broken after the changes I made, I need to mock `getAllowance` so in one case the token is considered enabled and in the other it's considered disabled. I'll make the update

Reply:
It's updated. I've created a `withEnabledToken` story so we can wrap individual stories with it to set a token as enabled